src/main/java/net/countercraft/movecraft/combat/movecraftcombat/status/StatusManager.java

```diff
@@ -100,31 +101,36 @@ public void craftReleased(@NotNull CraftReleaseEvent e) {
         if(event.isCancelled())
             return;
 
+        boolean manOverboard = canManOverboard(player, craft);
+
         if(Config.CombatReleaseScuttle) {
             player.sendMessage(ChatColor.RED + I18nSupport.getInternationalisedString("Combat Release Message"));
             e.setCancelled(true);
             craft.setNotificationPlayer(null);
             craft.sink();
         }
-        if(Config.CombatReleaseBanLength > 0) {
-            new BukkitRunnable() {
-                @Override
-                public void run() {
-                    Date expiry = new Date(System.currentTimeMillis() + Config.CombatReleaseBanLength * 1000);
-                    Bukkit.getServer().getBanList(BanList.Type.NAME).addBan(player.getName(), I18nSupport.getInternationalisedString("Combat Release"), expiry, "Movecraft-Combat AutoBan");
-                }
-            }.runTaskLater(MovecraftCombat.getInstance(), 5);
-        }
-        if(Config.CombatReleaseBanLength > 0 || Config.EnableCombatReleaseKick) {
-            new BukkitRunnable() {
-                @Override
-                public void run() {
-                    if(player == null || !player.isOnline()) {
-                        return;
+
+        if(manOverboard) {
```

Comment:
To simplify and streamline the code performance, let's change this to the following format:
```java
if(Config.CombatReleaseBanLength <= 0 && !Config.EnableCombatReleaseKick)
  return;
if(!canOverboard())
  return;
if(Config.CombatReleaseBanLength > 0) {
  // do ban
}
// do kick
```